recipe/meta.yaml

```diff
@@ -10,25 +10,29 @@ source:
   sha256: 412b92d3a28211dc5ea8191fc1d95832759e1f24548bc2e8d7c4c0d1d92b904c
 
 build:
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv
+  # pymc not available for py38
+  skip: True  # [py<39]
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
   number: 0
 
 requirements:
   host:
-    - python >=3.8
-    - setuptools >=61.0
+    - python
+    - setuptools >=61.0.0
     - pip
+    - wheel
   run:
-    - python >=3.8
+    - python
     - arviz >=0.13.0
     - matplotlib-base >=3.5.1
-    - numpy >=1.17
+    - numpy {{ numpy }}
     - pandas
-    - pymc >=5.3.0
+    - pymc >=5.5.0
     - scikit-learn >=1.1.1
     - seaborn >=0.12.2
     - xarray
+    - xarray-einstats >=0.5.1
+    - pymc-experimental >=0.0.8
```

Comment:
`pymc-experemental` is from the **main** branch made on **Jun 27** https://github.com/pymc-labs/pymc-marketing/blob/0855e0139c88719d2c3e16f59b5655a1a0df4115/pyproject.toml#L25 and not from **v0.1.1** https://github.com/pymc-labs/pymc-marketing/blob/0.1.1/pyproject.toml that has been released on **May 3** https://github.com/pymc-labs/pymc-marketing/releases/tag/0.1.1

Reply:
Oh you're right, thanks for catching that!